# Off Normal Weak
string value
---
int32 errorCode
string errorDescription